modules/gltf/src/lib/gltf-scenegraph.js

```diff
@@ -263,6 +265,29 @@ export default class GLTFScenegraph {
     }
   }
 
+  // Set default scene which is to be displayed at load time
+  defineDefaultScene(sceneIndex) {
```

Comment:
`setDefaultScene`

Reply:
Done in https://github.com/visgl/loaders.gl/pull/970